### PR TITLE
include: net: Cloud API extension

### PR DESCRIPTION
**Pull request by simensrostad**
Added endpoint topic used to publish batched data

### CODE REVIEW

File: include/net/cloud.h [hunk 1]
```diff
@@ -53,6 +53,7 @@ enum cloud_endpoint {
 	CLOUD_EP_TOPIC_STATE,
 	CLOUD_EP_TOPIC_CONFIG,
 	CLOUD_EP_TOPIC_PAIR,
+	CLOUD_EP_TOPIC_BATCH,
```
Comment (rlubos) [inline include/net/cloud.h]: Is this going to be used somewhere?

Comment (jtguggedal) [inline include/net/cloud.h]: I assume this is the topic that's intended for batch updates of some sorts? I'm not quite sure about the naming, but don't have any better suggestions either..

Comment (simensrostad) [reply include/net/cloud.h]: Yes, you are correct. It is currently used in the [cat tracker](https://github.com/bifravst/cat-tracker-fw) to publish buffered gps fixes in batches to a seperate topic, as per request from the frontend developer. I guess it makes it easier to seperate "historical" data from the latest sensor sample, which is typically published with CLOUD_EP_TOPIC_MSG.

Comment (rlubos) [reply include/net/cloud.h]: Thanks for explaining.

Comment (jtguggedal) [reply include/net/cloud.h]: All right, thanks.

**Pull request merged**